src/main/java/bio/terra/service/profile/ProfileDao.java

```diff
@@ -41,6 +41,8 @@
         + " FROM billing_profile"
         + " WHERE id in (:idlist)";
 
+    private static final String sqlListOld = "SELECT " + sqlSelectList
+        + " FROM billing_profile";
```

Comment:
not specific to this pr and def not a blocker---but dont we try to use templating for sql queries in our daos?

Reply:
We had talked about that. I don't know the pros and cons. My main goal here was to share the select list.

Reply:
I think this is OK since the string being constructed is a template string itself. The string doesn't contain any user input, only string constants defined in the current scope.